examples/messagecommands.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+
+import json
+
+from flask import Flask
+
+# This is just here for the sake of examples testing
+# to make sure that the imports work
+# (you don't actually need it in your code)
+sys.path.insert(1, ".")
+
+from flask_discord_interactions import DiscordInteractions  # noqa: E402
+from flask_discord_interactions.context import ApplicationCommandType
+
+
+app = Flask(__name__)
+discord = DiscordInteractions(app)
+
+# Find these in your Discord Developer Portal, store them as environment vars
+app.config["DISCORD_CLIENT_ID"] = os.environ["DISCORD_CLIENT_ID"]
+app.config["DISCORD_PUBLIC_KEY"] = os.environ["DISCORD_PUBLIC_KEY"]
+app.config["DISCORD_CLIENT_SECRET"] = os.environ["DISCORD_CLIENT_SECRET"]
+
+
+# Clear any existing global slash commands
+discord.update_slash_commands()
+
+
+# Simple command to change a message
+@discord.command(name="Make it bold", type=ApplicationCommandType.MESSAGE)
+def boldMessage(ctx):
+    return f"**{json.loads(json.dumps(next(iter(next(iter(ctx.resolved.values())).values()))))['content']}**"
```

Comment:
There's gotta be a better way to expose this functionality to the user. Maybe `ctx.target` could be a reference to the targeted `Message` or `Member`? I get that it's possible to find this information in the `ctx.resolved` field, but users shouldn't have to go digging for it themselves.

(Also, is there a reason you're dumping this to JSON and then immediately loading it?)

Reply:
Yes, you're right. I miss the `Context` class where you handle it. Both informations are in the `ctx.resolved` field so we can start to handle the `Message` and then add a `target` field to reference it.

(Just a mistake, I forgot to change the example)

I'm gonna working on it tu update the `Context` class. But this class is not similar to `Discord.py` ? Because it duplicate a lot of objects ? For example, if the user wants to send a DM to another with a `Command`, he has to deal with the API or import another library ?